### DISCUSSION TITLE
Removing gitpage url from google search results

### DISCUSSION THREAD

**Discussion by irrevocablesake** [community/community#23203] [Other Feature Feedback, Questions, & Ideas]
<p>Hello everyone,</p>
<p>So recently about like a month ago i redesigned my git pages website which also included deleting some pages from the website but the deleted pages were already crawled now even after deleting the web page for that particular url it’s still showing up in the google search results.</p>
<p>I waited for a month so that google might uncrawl it but no avail.</p>
<p>Please and Thank You</p>

**Comment by otter-computer** [comment, marked as answer]
<p>I don’t believe this is something you can directly control but you can request certain URLs be removed from Google’s search index using the “Remove outdated content tool” here:</p>
<p><a href="https://support.google.com/webmasters/answer/7041154?hl=en" class="onebox" target="_blank" rel="nofollow noopener">https://support.google.com/webmasters/answer/7041154?hl=en</a></p>
<p>If you’re the owner of the content and have it set up in Google’s search console then you can do the same here:</p>
<p><a href="https://search.google.com/search-console/removals" class="onebox" target="_blank" rel="nofollow noopener">https://search.google.com/search-console/removals</a></p>
<p>You may need to do the same for any other search engines that have crawled your content, I’d recommend searching for their removal tools and doing the same there.</p>

**Comment by irrevocablesake** [comment]
<p>Oh got it, Thanks for the reply</p>
